src/shared/check_outdated_data.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+int check_outdated_data(int maxTimeDiff, char *receivingData) {
```

Comment:
This `char*` could be a `const char*` please make it so.

Reply:
yes